woocommerce/woocommerce
# Utilize `post install` Script for Composer

### All Submissions:






### Changes proposed in this Pull Request:



This pull request adds `composer install` to the `postinstall` action for our plugins. During this work I also noticed that `prepare` scripts are executing on `install`, and as a consequence, every package is being built. This was previously necessary, but with our build orchestration tooling, we can rely on that build packages as needed. This should remove any ambiguity around having to run the `composer-install` executor, as well as speed up updates since they won't build packages anymore.

Closes #32652.

### How to test the changes in this Pull Request:

1. Remove all `node_modules` and `vendor` folders in your cloned directory
2. Run `pnpm install`, it should not build any packages. It will still build the core plugin, however, due to a separate install script in `composer.json`.

### Other information:




### FOR PR REVIEWER ONLY:
